postgresql-libpq.cabal

```diff
@@ -42,8 +42,7 @@ Custom-setup
 -- If true,  use pkg-config,  otherwise use the pg_config based build
 -- configuration
 Flag use-pkg-config
-  default: False
-  manual:  True
```

Comment:
make `manual: False` implicit

Reply:
Just to be sure: did you mean _explicit_? It is implicit already, because relies on default value.

Reply:
I want to it be explicitly written out, I never remember which is the default value.

Reply:
err, I did mean explicit, yes.

Reply:
Done